crates/weaver_resolver/data/multi-registry/custom_registry/custom_registry.yaml

```diff
@@ -28,4 +28,7 @@ groups:
       - ref: auction.name
         requirement_level: recommended
       - ref: error.type
-        requirement_level: required
\ No newline at end of file
+        requirement_level: required
+
+imports:
+  - metric_ref: metric.example_counter
```

Comment:
another branch of the slack discussion:
we probably want wildcard imports and cover attribute namespaces rather than groups.

E.g.
```yaml
imports:
  - attribute.db.*
  - attribute.server.*
  - attribute.error.*
  - attribute.http.*
  - attribute.url.*
```

syntax allows users to allowlist attributes in bulk without importing everything or explicitly allowing each attribute. 
Attribute groups are meaningless - they are grouping mechanism and we move attributes between them a lot.

there are not stability guarantees for attribute groups.

A similar syntax can be useful for bulk-importing metrics, entities, events

```yaml
imports:
- metric.db.*
- metric.http.*
- metric.system.*
- entity.host.*
- ...
```